src/include/uservarmanager.h

```diff
@@ -24,8 +33,14 @@ class DLLIMPORT UserVariableManager : public Mgr<UserVariableManager>
         wxString        m_ActiveSet;
         wxArrayString   m_Preempted;
 
+        UserVarManagerUI* m_ui;
```

Comment:
std::unique_ptr please.
And get your free space act together. Why are there two empty lines.

Reply:
fixed in bf2875c